ISSUES.md

```diff
@@ -0,0 +1,43 @@
+# **How to create an issue for ONI**
+
+Please help us to improve ONI! Create your issues found following the next steps:
+
+
+## **Create an Issue**
+
+To create an issue please go to [ONI Issues](https://github.com/Open-Network-Insight/open-network-insight/issues), review current open issues for any matches.
```

Comment:
i think this would be better in an unordered list:

## **Create an Issue**

To create an issue please go to [ONI Issues](https://github.com/Open-Network-Insight/open-network-insight/issues), 
* review current open issues for any matches.
* If there is a match, please feel free to comment on the issue.
* if you don't find a match, please continue creating your own issue.